ci-operator/step-registry/hypershift/kubevirt/baremetalds/conformance/hypershift-kubevirt-baremetalds-conformance-workflow.yaml

```diff
@@ -40,7 +41,7 @@ workflow:
       DEVSCRIPTS_CONFIG: |
         IP_STACK=v4
         NETWORK_TYPE=OVNKubernetes
-        NUM_WORKERS=4
+        NUM_WORKERS=2
         WORKER_VCPU=16
         WORKER_MEMORY=65536
```

Comment:
I think we need to give the master nodes a little more memory in this environment. can you set this env var `MASTER_MEMORY=20480`. That will bump the master node memory from 16Gi to 20Gi, which will hopefully improve the situation for us